yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
are you on the newest version of yarn (`1.19.1`)?

if so, it might be best if we all update so that we don't get a ping-ponging `yarn.lock` file where an older version removes this line and a newer version adds it on PRs. we might need to let everyone know to upgrade 🤔

Reply:
I think this was auto generated after a delete I performed. Gonna see if I can remove this version and merge master